src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <ProjectGuid>{15DB0DCC-68B4-4CFB-8BD2-F26BCCAF5A3F}</ProjectGuid>
     <IncludePartialFacadeTests Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">true</IncludePartialFacadeTests>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;uap-Debug;uap-Release</Configurations>
+    <TestRuntime>true</TestRuntime>
```

Comment:
Some like System.Memory test also have some code in a specific dll. We can either add it to buildtools, i.e.: TestRuntime adds System.Private.CoreLib, or explicitly here by adding `<CodeCoverageAssemblies>System.Memory</CodeCoverageAssemblies>`.  I think the later is better, I will do a PR for that on buildtools

Reply:
Edit the initial comment from "later" to "former"

Reply:
> Some like System.Memory test also have some code in a specific dll.

Right, it is the common pattern for `<TestRuntime>true</TestRuntime>` tests.

> TestRuntime adds System.Private.CoreLib,

Thanks - I though that it is the case already, but it is apparently not.